app_s2i/templates/includes/_jumbotron.html

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div style="background:transparent !important" class="jumbotron jumbotron-fluid">
-    <h1 class="display-4">Welcome to the DevOps Learning Community</h1>
+    <h1 class="display-5">Welcome to the DevOps Learning Community</h1>
```

Comment:
why this change was required. the jumbotron doesn't look good after it.

Reply:
In my resolution the jumbotron takes about 50%
<img width="956" alt="Annotation 2019-05-27 225837" src="https://user-images.githubusercontent.com/27818726/58437641-80de0700-80d3-11e9-8ad3-9d2f69145a6e.png">

Reply:
which is the purpose :-)

Reply:
It is useful in the home tab. 
Why to have the same 50% heading on ALL tabs?

Reply:
Maybe you are right. I don’t know. Do we want to address it in this PR?